pkg/utils/apiresources.go

```diff
@@ -177,16 +197,19 @@ func NewResourceConfig(isAllowList bool) *ResourceConfig {
 
 	// disable all fleet placement resources except for the envelope type
 	r.AddGroupKind(ClusterResourcePlacementGK)
```

Comment:
could you please add a task or TODO? we need to support group instead of listing all the apis.